# Libgda Dutch Translation.
# Copyright (C) 2000 Free Software Foundation, Inc.
# Almer S. Tigelaar <almer1@dds.nl>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: Libgda CVS\n"
"POT-Creation-Date: 2000-10-17 20:22+0200\n"
"PO-Revision-Date: 2000-10-17 21:05+0200\n"
"Last-Translator: Almer S. Tigelaar <almer1@dds.nl>\n"
"Language-Team: Dutch <nl@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:169
#, c-format
msgid "could not open connection to %s"
msgstr "Kon verbinding naar %s niet openen"

#: lib/gda-client/gda-connection-pool.c:180
#, c-format
msgid "Data source %s not found"
msgstr "Gegevensbron %s kon niet worden gevonden"

#: lib/gda-client/gda-connection.c:162 lib/gda-client/gda-error.c:116
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: De server gaf geen antwoord."

#: lib/gda-client/gda-connection.c:166 lib/gda-client/gda-error.c:120
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Een fout is opgetreden in het CORBA systeem."

#: lib/gda-client/gda-connection.c:449
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Databank '%s' niet gevonden in systeemconfiguratie"

#: lib/gda-client/gda-connection.c:465
msgid "Could not open CORBA factory"
msgstr "Kon 'CORBA factory' niet openen"

#: lib/gda-client/gda-field.c:187
msgid "<NULL>"
msgstr "<NUL>"

#: lib/gda-client/gda-field.c:193 lib/gda-client/gda-field.c:195
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Onbekend GDA Type(NUL)>"

#: lib/gda-client/gda-field.c:208
msgid "TRUE"
msgstr "WAAR"

#: lib/gda-client/gda-field.c:209
msgid "FALSE"
msgstr "ONWAAR"

#: lib/gda-common/gda-corba.c:80
msgid "True"
msgstr "Waar"

#: lib/gda-common/gda-corba.c:80
msgid "False"
msgstr "Onwaar"

#: lib/gda-common/gda-corba.c:127
#, c-format
msgid "CORBA System exception: %s"
msgstr "CORBA Systeem uitzonderings: %s"

#: lib/gda-common/gda-init.c:58
msgid "Attempt to initialize an already initialized client"
msgstr "Poging om een reeds geinitializeerde client opnieuw te initializeren"

#: lib/gda-common/gda-log.c:47
msgid "Saving log...x"
msgstr "Bezig met opslaan log...x"

#. compose message
#: lib/gda-common/gda-log.c:68
msgid "ERROR: "
msgstr "FOUT:"

#: lib/gda-common/gda-log.c:68
msgid "MESSAGE: "
msgstr "BERICHT:"

#: lib/gda-common/gda-thread.c:210
msgid "thread is already running"
msgstr "'thread' draait al"

#: lib/gda-common/gda-xml-database.c:281
#, c-format
msgid "table %s already exists"
msgstr "tabel %s bestaat al"

#: lib/gda-common/gda-xml-database.c:445
#, c-format
msgid "%p is not a valid table"
msgstr "%p is geen geldige tabel"

#: lib/gda-server/gda-server-error.c:169
msgid "Could not get pointer to server implementation"
msgstr "Kon 'pointer' naar de server implementatie niet vergaren"

#: lib/gda-server/gda-server-impl-connection.c:106
#: lib/gda-server/gda-server-impl-connection.c:115
#: lib/gda-server/gda-server-impl-connection.c:122
#: lib/gda-server/gda-server-impl-connection.c:131
#: lib/gda-server/gda-server-impl-connection.c:138
#: lib/gda-server/gda-server-impl-connection.c:147
#: lib/gda-server/gda-server-impl-connection.c:154
#: lib/gda-server/gda-server-impl-connection.c:163
#: lib/gda-server/gda-server-impl-recordset.c:113
#: lib/gda-server/gda-server-impl-recordset.c:122
#: lib/gda-server/gda-server-impl-recordset.c:129
#: lib/gda-server/gda-server-impl-recordset.c:144
#: lib/gda-server/gda-server-impl-recordset.c:153
#: lib/gda-server/gda-server-impl-recordset.c:160
#: lib/gda-server/gda-server-impl-recordset.c:169
#: lib/gda-server/gda-server-impl-recordset.c:176
#: lib/gda-server/gda-server-impl-recordset.c:185
#: lib/gda-server/gda-server-impl-recordset.c:192
#: lib/gda-server/gda-server-impl-recordset.c:200
#: lib/gda-server/gda-server-impl-recordset.c:209
#: lib/gda-server/gda-server-impl-recordset.c:216
#: lib/gda-server/gda-server-impl-recordset.c:224
#: lib/gda-server/gda-server-impl-recordset.c:232
#: lib/gda-server/gda-server-impl-recordset.c:274
#: lib/gda-server/gda-server-impl-recordset.c:284
#: lib/gda-server/gda-server-impl-recordset.c:292
#: lib/gda-server/gda-server-impl-recordset.c:300
#: lib/gda-server/gda-server-impl-recordset.c:308
#: lib/gda-server/gda-server-impl-recordset.c:316
#: lib/gda-server/gda-server-impl-recordset.c:325
#: lib/gda-server/gda-server-impl-recordset.c:333
#: lib/gda-server/gda-server-impl-recordset.c:341
#, c-format
msgid "%s not implemented"
msgstr "%s niet geimplementeerd"

#: lib/gda-server/gda-server-impl-connection.c:255
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_open: Bezig met openen verbinding met DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:240
#: lib/gda-server/gda-server-impl-recordset.c:248
#, c-format
msgid "%s: not implemented"
msgstr "%s: niet geimplementeerd"

#: lib/gda-server/gda-server-impl-recordset.c:434
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: een fout trad op tijdens het ophalen van gegevens"

#: lib/gda-server/gda-server-impl-recordset.c:473
#, c-format
msgid "%s: length = %d"
msgstr "%s: lengte = %d"

#: lib/gda-server/gda-server-impl-recordset.c:486
#, c-format
msgid "%s: sending field description for '%s'"
msgstr "%s: bezig met versturing veld omschrijving voor '%s'"

#: lib/gda-server/gda-server-impl.c:114
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Bezig met starten provider %s met geen implementatie functies"

#: lib/gda-server/gda-server-impl.c:141
#: report/server/gda-report-server-main.c:120
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr "Fout: OAF kent onze IID niet; dit geeft een foutieve installatie aan!"

#: lib/gda-server/gda-server-impl.c:144
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr "Fout: Een andere instantie van deze provider is al geregistreerd bij OAF."

#: lib/gda-server/gda-server-impl.c:147
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr "Fout: Een onbekende fout trad op tijdens het registreren van deze provider bij OAF."

#: lib/gda-server/gda-server-impl.c:151
#, c-format
msgid "Registered with ID = %s"
msgstr "Geregistreerd met ID = %s"

#: lib/gda-server/gda-server-impl.c:233
#, c-format
msgid "CORBA system exception %s"
msgstr "CORBA systeem uitzondering %s"

#: lib/gda-server/gda-server-impl.c:236
#, c-format
msgid "CORBA user exception: %s"
msgstr "CORBA gebruikers uitzondering: %s"

#: lib/gda-server/gda-server-init.c:39 lib/gda-server/gda-server-init.c:45
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Omdat signaal %d ontvangen is word nu afgesloten"

#: lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Signaal %d ontvangen, maar word genegeert"

#: lib/gda-server/gda-server-init.c:97
msgid "Attempt to initialize an already initialized provider"
msgstr "Poging om een reeds geinitialiseerde provider opnieuw te initialiseren"

#: providers/gda-interbase-server/gda-interbase-connection.c:252
#: providers/gda-ldap-server/gda-ldap-connection.c:263
#: providers/gda-mdb-server/gda-mdb-connection.c:351
#: providers/gda-mysql-server/gda-mysql-connection.c:378
#: providers/gda-odbc-server/gda-odbc-connection.c:600
#: providers/gda-odbc-server/gda-odbc-connection.c:633
#: providers/gda-oracle-server/gda-oracle-connection.c:417
#: providers/gda-postgres-server/gda-postgres-connection.c:771
#: providers/gda-sybase-server/gda-sybase-error.c:106
#, c-format
msgid "error '%s' at %s"
msgstr "fout '%s' bij %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:257
#: providers/gda-interbase-server/gda-interbase-connection.c:258
#: providers/gda-ldap-server/gda-ldap-connection.c:268
#: providers/gda-ldap-server/gda-ldap-connection.c:269
#: providers/gda-mdb-server/gda-mdb-connection.c:355
#: providers/gda-mdb-server/gda-mdb-connection.c:356
#: providers/gda-mysql-server/gda-mysql-connection.c:382
#: providers/gda-mysql-server/gda-mysql-connection.c:383
#: providers/gda-odbc-server/gda-odbc-connection.c:602
#: providers/gda-odbc-server/gda-odbc-connection.c:603
#: providers/gda-odbc-server/gda-odbc-connection.c:635
#: providers/gda-odbc-server/gda-odbc-connection.c:636
#: providers/gda-oracle-server/gda-oracle-connection.c:422
#: providers/gda-oracle-server/gda-oracle-connection.c:423
#: providers/gda-postgres-server/gda-postgres-connection.c:778
#: providers/gda-postgres-server/gda-postgres-connection.c:779
#: providers/gda-sybase-server/gda-sybase-error.c:115
#: providers/gda-sybase-server/gda-sybase-error.c:116
msgid "Not available"
msgstr "Niet beschikbaar"

#: providers/gda-interbase-server/gda-interbase-connection.c:259
#: providers/gda-ldap-server/gda-ldap-connection.c:270
#: providers/gda-mdb-server/gda-mdb-connection.c:357
#: providers/gda-mysql-server/gda-mysql-connection.c:384
#: providers/gda-oracle-server/gda-oracle-connection.c:424
#: providers/gda-postgres-server/gda-postgres-connection.c:780
#: providers/gda-sybase-server/gda-sybase-error.c:117
msgid "error"
msgstr "fout"

#: providers/gda-ldap-server/main-ldap.c:68
msgid "Could not register LDAP provider implementation"
msgstr "Kon LDAP provider implementatie niet registreren"

#: providers/gda-interbase-server/gda-interbase-connection.c:251
#, c-format
msgid "Error code %ld"
msgstr "Foutcode %ld"

#: providers/gda-interbase-server/main-interbase.c:69
msgid "Could not register INTERBASE provider implementation"
msgstr "Kon INTERBASE provider implementatie niet registreren "

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:251
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:511
#: providers/gda-sybase-server/gda-sybase-connection.c:253
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "Onafgehandeld SCHEMA_QTYPE %d"

#: providers/gda-mail-server/main-mail.c:69
msgid "Could not register Mail provider implementation"
msgstr "Kon Mail provider implementatie niet registreren"

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:87
#: providers/gda-sybase-server/gda-sybase-command.c:116
#, c-format
msgid "executing command '%s'"
msgstr "bezig met uitvoeren van commando '%s'"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:89
#, c-format
msgid "info '%s'"
msgstr "informatie '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:148
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr "%s:%d: Je kunt geen UNIX_SOCKET geven als je ook een HOST of POORT geeft. Verwijder de UNIX_SOCKET of verwijder zowel de HOST als POORT opties"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s:%d: Je moet de bestandsnaam geven met de MS Access databank"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Bezig met openen van verbinding met bestandsnaam=%s, gebruiker=%s, wachtwoord=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:406
#: providers/gda-mysql-server/gda-mysql-connection.c:433
msgid "mysql_recset is NULL"
msgstr "mysql_recset is NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:448
#: providers/gda-mysql-server/gda-mysql-connection.c:475
#: providers/gda-sybase-server/gda-sybase-connection.c:407
msgid "No table name given for SCHEMA_COLS"
msgstr "Geen tabelnaam gegeven voor SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:69
#: providers/gda-mysql-server/main-mysql.c:69
msgid "Could not register MySQL provider implementation"
msgstr "Kon MySQL provider implementatie niet registreren"

#: providers/gda-mysql-server/gda-mysql-connection.c:162
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, "
"unix_socket=%s"
msgstr "Bezig met openen van verbinding met gebruiker=%s, wachtwoord=%s, host=%s, poort=%s, unix_socket=%s"

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: Onafgehandeld SCHEMA_QTYPE %d\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:617
#: providers/gda-odbc-server/gda-odbc-connection.c:618
#: providers/gda-odbc-server/gda-odbc-connection.c:619
#: providers/gda-odbc-server/gda-odbc-connection.c:650
#: providers/gda-odbc-server/gda-odbc-connection.c:651
#: providers/gda-odbc-server/gda-odbc-connection.c:652
msgid "no text"
msgstr "geen tekst"

#: providers/gda-odbc-server/main-odbc.c:68
msgid "Could not register ODBC provider implementation"
msgstr "Kon ODBC provider implementatie niet registreren"

#: providers/gda-oracle-server/gda-oracle-command.c:195
msgid "Could not allocate statement handle"
msgstr "Kon 'statement handle' niet toewijzen"

#: providers/gda-oracle-server/gda-oracle-command.c:199
msgid "command object being created without associated connection"
msgstr "commando object word gemaakt zonder bijbehorende verbinding"

#: providers/gda-oracle-server/gda-oracle-command.c:226
#, c-format
msgid "Executing command '%s'"
msgstr "Bezig met uitvoeren van commando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "fout in OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "fout in OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "fout in OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:370
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Onbekend ORACLE type %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Onbekend ORACLE gegevenstype in 'fill_field_values'"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr "recordset object word aangemaakt zonder bijbehorende verbinding"

#: providers/gda-oracle-server/main-oracle.c:68
msgid "Could not register ORACLE provider implementation"
msgstr "Kon ORACLE provider implementatie niet registreren"

#: providers/gda-postgres-server/gda-postgres-connection.c:880
#: providers/gda-postgres-server/gda-postgres-connection.c:992
msgid "postgres_recset is NULL"
msgstr "postgres_recset is NULL"

#: providers/gda-postgres-server/main-postgres.c:68
msgid "Could not register PostgreSQL provider implementation"
msgstr "Kon PostgreSQL provider implementatie niet registreren"

#: providers/gda-sybase-server/gda-sybase-command.c:111
msgid "Connection seems to be dead"
msgstr "Het lijkt erop dat de verbinding dood is"

#: providers/gda-sybase-server/gda-sybase-command.c:131
msgid "Filling command structure failed\n"
msgstr "Opvullen van commando structuur mislukt\n"

#: providers/gda-sybase-server/gda-sybase-command.c:143
msgid "sending cmd failed"
msgstr "verzenden van commando mislukt"

#: providers/gda-sybase-server/gda-sybase-command.c:150
msgid "Allocating recordset failed"
msgstr "Toewijzen van recordset mislukt"

#: providers/gda-sybase-server/gda-sybase-command.c:160
msgid "Getting recordset userdata failed"
msgstr "Ophalen van recordset gebruikers gegevens mislukt"

#: providers/gda-sybase-server/gda-sybase-command.c:218
msgid "Command execution finished, all results processed"
msgstr "Commando uitvoering klaar, alle resultaten zijn verwerkt."

#: providers/gda-sybase-server/gda-sybase-command.c:222
msgid "Command failed"
msgstr "Commando mislukt"

#: providers/gda-sybase-server/gda-sybase-command.c:226
msgid "Execution of resultless command succeeded"
msgstr "Uitvoering van resultaatloos commando gelukt"

#: providers/gda-sybase-server/gda-sybase-command.c:242
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() gaf %x terug"

#: providers/gda-sybase-server/gda-sybase-command.c:252
msgid "No further results, command completly succeeded."
msgstr "Geen verdere resultaten, commando is volledig gelukt."

#: providers/gda-sybase-server/gda-sybase-command.c:256
msgid "An error occured during command execution."
msgstr "Een fout trad op tijdens het uitvoeren van het commando."

#: providers/gda-sybase-server/gda-sybase-command.c:263
msgid "Cmd execution returned unknown result, may be an error"
msgstr "Commando uitvoering leverde een onbekend resultaat op, dit zou op een fout kunnen duiden"

#: providers/gda-sybase-server/gda-sybase-command.c:276
msgid "No data returned, freeing recordset structure"
msgstr "Geen gegevens teruggegeven, bezig met het vrijgeven van recordset structuur"

#: providers/gda-sybase-server/gda-sybase-command.c:307
#: providers/gda-sybase-server/gda-sybase-command.c:319
msgid "Freeing command structure failed"
msgstr "Vrijgeven van recordset structuur mislukt"

#: providers/gda-sybase-server/gda-sybase-connection.c:133
msgid "Error allocating error structure"
msgstr "Fout bij het toewijzen van fout structuur"

#: providers/gda-sybase-server/gda-sybase-connection.c:141
msgid "Not enough memory for sybase_Connection"
msgstr "Niet genoeg geheugen voor sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:397
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: tabelnaam '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:413
msgid "Could not allocate command structure"
msgstr "Kon commando structuur niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:435
msgid "Could not allocate enough memory for query"
msgstr "Kon niet genoeg geheugen toewijzen voor de query"

#: providers/gda-sybase-server/gda-sybase-connection.c:449
msgid "Could not get recordset in schema_cols"
msgstr "Kon recordset in schema_cols niet vergaren"

#: providers/gda-sybase-server/gda-sybase-connection.c:481
#: providers/gda-sybase-server/gda-sybase-connection.c:558
#, c-format
msgid "schema_tables: table name = '%s'"
msgstr "schema_tables: tabelnaam = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:489
#: providers/gda-sybase-server/gda-sybase-connection.c:566
#, c-format
msgid "schema_tables: table_schema = '%s'"
msgstr "schema_tables: tabelschema = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:494
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
msgstr "schema_procedures: proc_catlog = '%s' ONGEBRUIKT!\n"

#: providers/gda-sybase-server/gda-sybase-connection.c:498
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: ongeldig beperkt type %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:508
msgid "Could not allocate cmd structure"
msgstr "Kon cmd structuur niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:571
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
msgstr "schema_procedures: proc_catalog = '%s' ONGEBRUIKT!"

#: providers/gda-sybase-server/gda-sybase-connection.c:575
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tabels: ongeldig beperkt type %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:636
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Niet ondersteund of ongeldig beperkt type: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:677
msgid "Could not allocate Gda_ServerCommand"
msgstr "Kon Gda_ServerCommand niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:707
#: providers/gda-sybase-server/gda-sybase-connection.c:712
#, c-format
msgid "connection status: %x"
msgstr "verbindings status: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:708
#, c-format
msgid "return code: %x"
msgstr "resultaat code: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:745
msgid "attempting reconnection"
msgstr "bezig met opnieuw verbinden"

#: providers/gda-sybase-server/gda-sybase-connection.c:884
#, c-format
msgid "Reconnected to '%s'"
msgstr "Opnieuw verbonden met '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:886
#, c-format
msgid "Connected to %s"
msgstr "Verbonden met %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:894
#, c-format
msgid "Selected database '%s'"
msgstr "Geselecteerde databank '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:923
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Kon cmd structuur niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:932
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Kon geheugen for cmd tekenreeks niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:945
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command mislukt"

#: providers/gda-sybase-server/gda-sybase-connection.c:954
#, c-format
msgid "%s: sending command failed"
msgstr "%s: versturen van commando mislukt"

#: providers/gda-sybase-server/gda-sybase-connection.c:965
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: selecteren van databank mislukt"

#: providers/gda-sybase-server/gda-sybase-connection.c:983
#, c-format
msgid "%s: use command failed"
msgstr "%s: 'use' commando mislukt"

#: providers/gda-sybase-server/gda-sybase-connection.c:1010
msgid "Could not allocate locale structure"
msgstr "Kon lokale structuur niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1016
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Kon lokale instellingen niet veranderen naar '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1022
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Kon lokale instellingen niet configureren naar '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1027
msgid "Could not drop locale structure"
msgstr "Kon lokale structuur niet verwijderen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1040
#, c-format
msgid "Fatal error: %s\n"
msgstr "Fatale fout: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:62
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:71
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr "Er is geen fout vlag ingesteld, toch werd %s aangeroepen. Dit zou een fout in het programma kunnen zijn."

#: providers/gda-sybase-server/gda-sybase-error.c:135
msgid "cs_diag() initialized"
msgstr "cd_diag() geinitializeerd"

#: providers/gda-sybase-server/gda-sybase-error.c:138
msgid "Could not initialize cs_diag()"
msgstr "Kon cs_diag() niet initializeren"

#: providers/gda-sybase-server/gda-sybase-error.c:145
msgid "ct_diag() initialized"
msgstr "ct_diag() geinitializeerd"

#: providers/gda-sybase-server/gda-sybase-error.c:148
msgid "Could not initialize ct_diag()"
msgstr "Kon ct_diag() niet initializeren"

#: providers/gda-sybase-server/gda-sybase-error.c:182
#, c-format
msgid "%s could not find server implementation"
msgstr "%s kon server implementatie niet vinden"

#: providers/gda-sybase-server/gda-sybase-error.c:188
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s kon cnc gebruikers gegevens niet vergaren"

#: providers/gda-sybase-server/gda-sybase-error.c:200
msgid "Could not get # of messages"
msgstr "Kon # berichten niet vergaren"

#: providers/gda-sybase-server/gda-sybase-error.c:205
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d ctlib berichten om op te halen"

#: providers/gda-sybase-server/gda-sybase-error.c:207
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d ctlib bericht om op te halen"

#: providers/gda-sybase-server/gda-sybase-error.c:243
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "fout is een 'nullpointer' in gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:268
#: providers/gda-sybase-server/gda-sybase-error.c:335
#: providers/gda-sybase-server/gda-sybase-error.c:378
msgid "Could not get # of server messages"
msgstr "Kon # server berichten niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:276
#, c-format
msgid "Fetching %d client messages"
msgstr "Bezig met ophalen %d client berichten"

#: providers/gda-sybase-server/gda-sybase-error.c:278
#, c-format
msgid "Fetching %d client message"
msgstr "Bezig met ophalen %d client bericht"

#: providers/gda-sybase-server/gda-sybase-error.c:287
#, c-format
msgid "Could not get client msg # %d"
msgstr "Kon # %d client berichten niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:303
msgid "Could not get # of client messages"
msgstr "Kon # client berichten niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:311
msgid "Could not clear client messages"
msgstr "Kon client berichten niet leegmaken"

#: providers/gda-sybase-server/gda-sybase-error.c:343
#, c-format
msgid "Fetching %d server messages"
msgstr "Bezig met ophalen %d server berichten"

#: providers/gda-sybase-server/gda-sybase-error.c:345
#, c-format
msgid "Fetching %d server message"
msgstr "Beizg met ophalen %d server bericht"

#: providers/gda-sybase-server/gda-sybase-error.c:354
#, c-format
msgid "Could not get server msg # %d"
msgstr "Kon server bericht # %d niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:386
msgid "Could not clear server messages"
msgstr "Kon server berichten niet leegmaken"

#: providers/gda-sybase-server/gda-sybase-error.c:408
#: providers/gda-sybase-server/gda-sybase-error.c:444
msgid "Could not get # of cslib messages"
msgstr "Kon # cslib berichten niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:416
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Bezg met ophalen %d cslib berichten"

#: providers/gda-sybase-server/gda-sybase-error.c:418
#, c-format
msgid "Fetchong %d cslib message"
msgstr "Bezig met ophalen %d cslib bericht"

#: providers/gda-sybase-server/gda-sybase-error.c:428
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Kon cslib bericht # %d niet ophalen"

#: providers/gda-sybase-server/gda-sybase-error.c:452
msgid "Could not clear cslib messages"
msgstr "Kon cslib berichten niet leegmaken"

#: providers/gda-sybase-server/gda-sybase-error.c:584
msgid "Server message:"
msgstr "Server bericht:"

#: providers/gda-sybase-server/gda-sybase-error.c:593
msgid "Client message:"
msgstr "Client bericht:"

#: providers/gda-sybase-server/gda-sybase-error.c:600
msgid "CS-Library error:"
msgstr "CS-Bibliotheek fout:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:152
#, c-format
msgid "fetch error at row %d"
msgstr "Ophaalfout bij rij %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:159
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA op rij %d; bezig met afbreken"

#: providers/gda-sybase-server/gda-sybase-recordset.c:168
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Fout (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:343
msgid "Failed fetching # of columns"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:349
#, c-format
msgid "Counted %d columns"
msgstr "%d kolommen geteld"

#: providers/gda-sybase-server/gda-sybase-recordset.c:356
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s kon gegevensformaat info niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:366
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s kon gegevensvelden niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:384
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe op kolom # %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:391
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe op kolom # %d mislukt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:397
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:406
msgid "could not allocate data holder"
msgstr "kon gegevenshouder niet toewijzen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:416
msgid "could not ct_bind data holder to column"
msgstr "kon gegevenshouder niet verbinden (ct_bind) met kolom"

#: providers/gda-sybase-server/gda-sybase-recordset.c:423
msgid "illegal type detected, aborting"
msgstr "illegaal type gedetecteerd, bezig met afbreken"

#: providers/gda-sybase-server/gda-sybase-recordset.c:429
msgid "could not allocate field"
msgstr "kon veld niet toewijzen"

#: providers/gda-sybase-server/main-sybase.c:73
msgid "Could not register gda-sybase provider implementation"
msgstr "Kon gda-sybase provider implementatie niet registreren"

#: report/gda-report-engine.c:71
#, c-format
msgid "CORBA exception unloading report engine: %s"
msgstr "CORBA uitzondering ontladen rapport engine: %s"

#: report/server/gda-report-server-main.c:43
#: report/server/gda-report-server-main.c:50
#, c-format
msgid "Received signal %d, shutting down."
msgstr "Als gevolg van ontvangst van signaal %d word nu afgesloten."

#: report/server/gda-report-server-main.c:54
#, c-format
msgid "Received signal %d, shutting down cleanly"
msgstr "Als gevolg van ontvangst van signaal %d word nu netjes afgesloten"

#: report/server/gda-report-server-main.c:96
msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
msgstr "Initializeren van OAF mislukt: Stuur a.u.b. een bug rapport naar de OAF beheerders"

#: report/server/gda-report-server-main.c:106
msgid "Could not activate report engine"
msgstr "Kon rapporteer engine niet activeren"

#: report/server/gda-report-server-main.c:123
msgid ""
"Another instance of the report engine is already registered with OAF. "
"Exiting..."
msgstr "Een andere instantie van de rapport engine is al geregistreerd bij OAF."

#: report/server/gda-report-server-main.c:126
msgid "Unknown error registering report engine with OAF. Exiting"
msgstr "Onbekende tijdens het registreren van de rapport engine bij OAF."

#: report/server/gda-report-server-main.c:130
#, c-format
msgid "Registered report engine with ID = %s"
msgstr "Raport engine geregistreerd met ID = %s"

#: tools/gda-run.c:29
msgid "database name"
msgstr "databank naam"

#: tools/gda-run.c:29
msgid "DATABASE"
msgstr "DATABANK"

#: tools/gda-run.c:30
msgid "user name"
msgstr "gebruikersnaam"

#: tools/gda-run.c:30
msgid "USER-NAME"
msgstr "GEBRUIKERSNAAM"

#: tools/gda-run.c:31
msgid "password"
msgstr "wachtwoord"

#: tools/gda-run.c:31
msgid "PASSWORD"
msgstr "WACHTWOORD"

#: tools/gda-run.c:32
msgid "file name"
msgstr "bestandsnaam"

#: tools/gda-run.c:32
msgid "FILE-NAME"
msgstr "BESTANDSNAAM"

#: tools/gda-run.c:104
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: verkeerd geconfigureerd DSN item"

#: tools/gda-run.c:119
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: gegevensbron '%s' kon niet worden gevonden\n"
